modules/ocf_mail/files/spam/logging/examine-mail-log

```diff
@@ -22,6 +23,12 @@ USER_EXCEPTIONS = {'root', 'apache', 'wordpress'}
 SYSTEM_USERS = {'ocfweb'}
 
 
+# vhost to user dictionary
+vhost_user_for_domain = {}
+for _vhost in get_mail_vhosts():
+    vhost_user_for_domain[_vhost.domain] = _vhost.user
```

Comment:
i'd rather not do this at the top level, imports should be as free as possible and this is slow